www/index.php

```diff
@@ -14,9 +14,14 @@
 
 $config = Configuration::get_configuration();
 $app = new Slim();
+$app->config('debug', true);
```

Comment:
I think just remove it.  I don't see a strong need to be opinionated about if you run debug mode or not in dev.  This did spark half a thought as to not running debug mode in production, but again, I don't think we need to express an opinion about that here.